tests/test_permissions_records.py

```diff
@@ -29,23 +32,33 @@
 }
 
 
-def test_record_permission(app):
+def test_record_list_permission_factory(app):
+    list_perm = record_list_permission_factory()
+
+    # Loading permissions in invenio-access always add superuser
+    assert list_perm.needs == {superuser_access, any_user}
+    assert list_perm.excludes == set()
+    assert list_perm.query_filters == [Q('match_all')]
+
 
+def test_record_create_permission_factory(app):
     create_perm = record_create_permission_factory(record)
-    list_perm = record_list_permission_factory()  # No record needed
-    read_perm = record_read_permission_factory(record)
-    read_files_perm = record_read_files_permission_factory(record)
-    update_perm = record_update_permission_factory(record)
-    delete_perm = record_delete_permission_factory(record)
 
     assert create_perm.needs == {superuser_access}
-    # FIXME: will fail because invenio-access adds all in 'needs'
+    # Fails because of bug in invenio-access:
```

Comment:
Should be okay after: https://github.com/inveniosoftware/invenio-access/pull/166